packages/blade/src/components/Accordion/AccordionButton.native.tsx

```diff
@@ -73,11 +73,16 @@ const _AccordionButton = ({ index, icon: Icon, children }: AccordionButtonProps)
     isPressed.value = pressed;
 
     const iconColor: IconProps['color'] = pressed
-      ? 'surface.action.icon.focus.lowContrast'
-      : 'surface.action.icon.default.lowContrast';
+      ? 'interactive.icon.gray.subtle'
```

Comment:
why are we using interactive and not surface?

Reply:
Following design here. It should be interactive since its the icon on accordion which is interactable